aries_cloudagent/protocols/connections/v1_0/routes.py

```diff
@@ -73,6 +73,7 @@ class InvitationConnectionTargetRequestSchema(OpenAPISchema):
         required=False,
         description="List of routing keys",
     )
+    mediation_id = UUIDFour
```

Comment:
Looks wrong: something like `fields.Str(required=False, description="...", **UUID4)` if mediation id must be a uuid4, `fields.Str(required=False, description="...", example=UUIDFour.EXAMPLE)` if it is typically but not necessarily a uuid4

Reply:
I think I understand, let me see if I can update this open API validation from your suggestion.